### ISSUE TITLE
Feature Request: Support Non-Failing Accessibility Tests for Report-Only Mode in CI/CD

### ISSUE THREAD

**Issue by hitshiroya** [component-driven/cypress-axe#185]
I have a use case where I'm running accessibility checks across multiple pages and collecting the violation data to build a report. The goal is to generate insights.

```
it("Accessibility test for page", () => {
  const pageName = "Some Page";
  cy.visit("/some-page");
  cy.injectAxe();
  
  cy.checkA11y(null, { failOnError: false }, (violations) => {
    cy.task("saveA11yViolations", { pageName, violations });
  });
});

```
The cy.task("saveA11yViolations") function stores violations in a JSON file to later build an accessibility report.

**Problem in CI/CD**
When I run this in a CI/CD pipeline, the test sometimes fails, which causes:

- The job to stop prematurely
- Artifacts (like the report JSON) to never be uploaded
- The entire pipeline to fail, even though the goal was just data collection

Can we add an official mode or configuration for use cases like this — where:
- The test is used only to gather and log violations
- The test should never fail, even if violations are found
- The pipeline should continue, ensuring logs and artifacts are always available

Or could you please let me know if something like this already exists and I’m just missing it?

**Environment**
cypress: 14.0.0

cypress-axe: 1.6.0

CI Platform: GitLab CI/CD

Thanks,
Hit

**Comment by hitshiroya** [comment]
Added a patch for the feature. Awaiting review — thanks!
https://github.com/component-driven/cypress-axe/pull/186
